### ISSUE TITLE
problems with setup.cfg tag_prefix = ""

### ISSUE THREAD

**Issue by ldath** [warner/python-versioneer#84]
When I have in my `setup.cfg` file:
```
[versioneer]
VCS = git
style = pep440
tag_prefix = ""
...
```
then in Python 2 I have `cfg.tag_prefix == ''` and in Python 3 `cfg.tag_prefix == '""'`. Solution is to make `setup.cfg` looks like:
```
[versioneer]
VCS = git
style = pep440
tag_prefix = 
...
```
Then in Python 2 and Python 3 `cfg.tag_prefix == ''`

**Comment by warner** [comment]
This should be fixed by #81, which changes the docs and sample config to use single quotes instead of double quotes.

Please check the comment on #81 for an additional potential fix.

**Comment by warner** [comment]
I pushed an extra fix in 2110827 to accept three different ways to express an empty prefix: `tag_prefix=`, `tag_prefix = ''`, and `tag_prefix = ""`.
